src/main/resources/css/scrollbar.css

```diff
@@ -0,0 +1,70 @@
+.scroll-pane {
```

Comment:
Why are you using decimals in this stylesheet?

Reply:
In Stack Overflow, I saw that they were using decimals, and I thought maybe that's the practice. I removed them.